userspace/falco/configuration.cpp

```diff
@@ -82,7 +82,10 @@ falco_configuration::falco_configuration():
 	m_metrics_output_file(""),
 	m_metrics_flags(0),
 	m_metrics_convert_memory_to_mb(true),
-	m_metrics_include_empty_values(false)
+	m_metrics_include_empty_values(false),
+	m_container_engines_mask(0),
+	m_container_engines_cri_socket_paths({"/run/containerd/containerd.sock", "/run/crio/crio.sock","/run/k3s/containerd/containerd.sock"}),
```

Comment:
Not for now but i'd expose these defaults as static strings from libs cri namespace, like: `libsinsp::container_engines::cri::s_default_socket_paths`.